src/eventsub/mod.rs

```diff
@@ -590,6 +634,15 @@ pub struct WebhookTransportResponse {
     pub callback: String,
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[cfg_attr(feature = "deny_unknown_fields", serde(deny_unknown_fields))]
+#[non_exhaustive]
+/// Conduit transport
+pub struct ConduitTransportResponse {
+    /// The conduit ID
+    pub conduit_id: String,
```

Comment:
this could arguably be a `twitch_types::ConduitId`, but we haven't done that for websockets and I feel like it's not really needed here